### PR TITLE
ADEN-8673 Build AE3 assets on sandbox build

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: skins/oasis/css/core/ads.scss [hunk 1]
```diff
@@ -1,4 +1,9 @@
+// TODO: Remove dist/ads.scss in the next release
```
Comment (jacekbj) [inline skins/oasis/css/core/ads.scss:1]: Do we have a ticket for that TODO?

File: docker/base/Dockerfile-php [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# This is a base Docker image used in prod/sandbox/preview Jenkinsfile
+# This is a base Docker image used in prod/preview Jenkinsfile
```
Comment (nandy-andy) [inline docker/base/Dockerfile-php:1]: 💎

File: skins/oasis/css/core/ads.scss (hunk 1 above)
Comment (fraszczakszymon) [reply skins/oasis/css/core/ads.scss:1]: ADEN-8690

File: extensions/wikia/AdEngine3/alert.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+require(['ext.wikia.adEngine3'], function () {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine3/alert.js:1]: ❤️

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**